src/main/java/org/myrobotlab/inmoov/Utils.java

```diff
@@ -16,13 +16,9 @@
   /**
    * This method will load a python file into the python interpreter. 
    */
-  public static boolean loadPythonFile(String file, String intanceName) {
+  public static boolean loadPythonFile(String file) {

```

Comment:
Also, this method should not return true if the underlying script execution had an error.. it should return false.  otherwise what's the point of returning anything?